### PR TITLE
Fix: Avoid displaying help messages in red

### PR DESCRIPTION
**Pull request by rchavik**
Fix some issues I found when running core tests in `APP/vendor/cakephp/cakephp`

See #11179

### CODE REVIEW

File: src/Console/Shell.php [hunk 1]
```diff
@@ -508,8 +508,7 @@ public function runCommand($argv, $autoMethod = false, $extra = [])
             return $this->main(...$this->args);
         }
 
-        $this->err('No subcommand provided. Choose one of the available subcommands.', 2);
```
Comment (dereuromark) [inline src/Console/Shell.php]: This should be in sync with the new command runner, thats why this line should probably not be removed, right?

Comment (rchavik) [reply src/Console/Shell.php]: I dunno. I just reverted the commit, so it wasn't there before 11179

Comment (markstory) [reply src/Console/Shell.php]: I think we should keep this as well.

**Comment by rchavik** [comment]
I've made the adjustment to keep the one line.

**Comment by thaJeztah** [comment]
Thanks!

**Pull request merged**